forge/tests/error/properties_undirected_tree_error.rkt

```diff
@@ -0,0 +1,42 @@
+#lang forge
```

Comment:
Error tests need to be registered in `forge/tests/error/main.rkt` with an expectation.

To check if it's all working, run the `main.rkt` file. (You can comment out the other tests to speed it up.)